TravelAgent/TravelAgent/Controllers/EmployeeTravelController.cs

```diff
@@ -19,13 +19,15 @@ public class EmployeeTravelController : ControllerBase
         private readonly IEmployeeRepository _employeeRepository;
         private readonly ITravelRepository _travelRepository;
         private readonly IApartmentRepository _apartmentRepository;
+        private readonly IOfficeRepository _officeRepository;
 
-        public EmployeeTravelController(IEmployeeTravelRepository employeeTravelRepository, IEmployeeRepository employeeRepository, ITravelRepository travelRepository, IApartmentRepository apartmentRepository)
+        public EmployeeTravelController(IEmployeeTravelRepository employeeTravelRepository, IEmployeeRepository employeeRepository, ITravelRepository travelRepository, IApartmentRepository apartmentRepository, IOfficeRepository officeRepository)
```

Comment:
You should really split these into multiple lines 👀

Reply:
This line be like 😄 
![image](https://user-images.githubusercontent.com/20258194/58429578-04d1c800-80af-11e9-939e-6ef3b963e935.png)